<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>900</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>11</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>lab_06. РЕАЛИЗАЦИЯ И ИССЛЕДОВАНИЕ АЛГОРИТМА ПОСТРОЧНОГО ЗАТРАВОЧНОГО  ЗАПОЛНЕНИЯ СПЛОШНЫХ ОБЛАСТЕЙ.</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(56, 56, 56);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0">
      <property name="spacing">
       <number>7</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="set_x_label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(200, 200, 200);
font: 11pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Координата X точки</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="set_x">
            <property name="styleSheet">
             <string notr="true">font: 11pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="set_y_label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(200, 200, 200);
font: 11pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Координата Y точки</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="set_y">
            <property name="styleSheet">
             <string notr="true">font: 11pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="delay_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(200, 200, 200);
font: 11pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Задержка:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="set_delay_cb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="add_point_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>460</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(200, 200, 200);
font: 10pt &quot;Segoe UI&quot;;
background-color: rgb(86, 86, 86);</string>
        </property>
        <property name="text">
         <string>Добавить точку</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="remove_point_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>460</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(200, 200, 200);
font: 10pt &quot;Segoe UI&quot;;
background-color: rgb(86, 86, 86);</string>
        </property>
        <property name="text">
         <string>Удалить точку</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add_zt_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>460</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(200, 200, 200);
font: 10pt &quot;Segoe UI&quot;;
background-color: rgb(86, 86, 86);</string>
        </property>
        <property name="text">
         <string>Добавить затравку</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="draw_limit_figure_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(200, 200, 200);
font: 11pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Построить ограничивающую фигуру</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="set_circle_figure">
          <property name="styleSheet">
           <string notr="true">color: rgb(200, 200, 200);
font: 11pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Окружность</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="set_ellipse_figure">
          <property name="styleSheet">
           <string notr="true">color: rgb(200, 200, 200);
font: 11pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Эллипс</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>130</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <widget class="QWidget" name="horizontalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>401</width>
            <height>67</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QLabel" name="set_xc_circle_label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(200, 200, 200);
font: 11pt &quot;Segoe UI&quot;;</string>
               </property>
               <property name="text">
                <string>x центра</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="set_xc_circle">
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QLabel" name="set_yc_circle_label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(200, 200, 200);
font: 11pt &quot;Segoe UI&quot;;</string>
               </property>
               <property name="text">
                <string>y центра</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="set_yc_circle">
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>70</y>
            <width>401</width>
            <height>51</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="set_radius_label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(200, 200, 200);
font: 11pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Радиус окружности</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="set_r_circle">
             <property name="styleSheet">
              <string notr="true">font: 11pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2">
         <widget class="QWidget" name="horizontalLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>2</y>
            <width>401</width>
            <height>61</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <widget class="QLabel" name="set_xc_ellipse_label">
               <property name="styleSheet">
                <string notr="true">color: rgb(200, 200, 200);
font: 11pt &quot;Segoe UI&quot;;</string>
               </property>
               <property name="text">
                <string>x центра</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="set_xc_ellipse">
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <widget class="QLabel" name="set_yc_ellipse_label">
               <property name="styleSheet">
                <string notr="true">color: rgb(200, 200, 200);
font: 11pt &quot;Segoe UI&quot;;</string>
               </property>
               <property name="text">
                <string>y центра</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="set_yc_ellipse">
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>70</y>
            <width>401</width>
            <height>61</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <widget class="QLabel" name="set_ellipse_width_label">
               <property name="styleSheet">
                <string notr="true">color: rgb(200, 200, 200);
font: 11pt &quot;Segoe UI&quot;;</string>
               </property>
               <property name="text">
                <string>Ширина</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="set_ellipse_width">
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_13">
             <item>
              <widget class="QLabel" name="set_ellipse_height_label">
               <property name="styleSheet">
                <string notr="true">color: rgb(200, 200, 200);
font: 11pt &quot;Segoe UI&quot;;</string>
               </property>
               <property name="text">
                <string>Высота</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="set_ellipse_height">
               <property name="styleSheet">
                <string notr="true">font: 11pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="change_color_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>460</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(200, 200, 200);
font: 10pt &quot;Segoe UI&quot;;
background-color: rgb(86, 86, 86);</string>
        </property>
        <property name="text">
         <string>Изменить цвет закраски</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="change_edge_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>460</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(200, 200, 200);
font: 10pt &quot;Segoe UI&quot;;
background-color: rgb(86, 86, 86);</string>
        </property>
        <property name="text">
         <string>Изменить цвет границы</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="close_figure_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>460</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(200, 200, 200);
font: 10pt &quot;Segoe UI&quot;;
background-color: rgb(86, 86, 86);</string>
        </property>
        <property name="text">
         <string>Замкнуть фигуру</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="paint_figure_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>460</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(200, 200, 200);
font: 10pt &quot;Segoe UI&quot;;
background-color: rgb(86, 86, 86);</string>
        </property>
        <property name="text">
         <string>Закрасить фигуру</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clear_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>460</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(200, 200, 200);
font: 10pt &quot;Segoe UI&quot;;
background-color: rgb(86, 86, 86);</string>
        </property>
        <property name="text">
         <string>Очистить все</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="scroll_list">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="current_coords_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(200, 200, 200);
font: 10pt &quot;Segoe UI&quot;;
border-color: rgb(255, 255, 255);
background-color: rgb(75, 75, 75);</string>
          </property>
          <property name="text">
           <string>x, y:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="current_grid_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>230</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(200, 200, 200);
font: 10pt &quot;Segoe UI&quot;;
border-color: rgb(255, 255, 255);
background-color: rgb(75, 75, 75);</string>
          </property>
          <property name="text">
           <string>Шаг сетки:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>100</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1600</width>
        <height>900</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1069</width>
     <height>29</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(116, 116, 116);</string>
   </property>
   <widget class="QMenu" name="about_menu">
    <property name="title">
     <string>О программе</string>
    </property>
    <addaction name="about_author"/>
    <addaction name="about_task"/>
    <addaction name="instruction"/>
   </widget>
   <addaction name="about_menu"/>
  </widget>
  <action name="about_author">
   <property name="text">
    <string>Об авторе</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="about_task">
   <property name="text">
    <string>Условие</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="instruction">
   <property name="text">
    <string>Возможности программы</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
